app/ide-desktop/common/src/services/Backend.ts

```diff
@@ -505,6 +505,21 @@ export interface CreateCustomerPortalSessionResponse {
   readonly url: string | null
 }
 
+/** Whether the user is in an organization. */
+export function isUserInOrganization(user: User) {
```

Comment:
The name is misleading as it checks if a user has a subscription, but having a subscription is not sufficient to be in organization, e.g. when the user cancelled it